src/MessagingInput/MessagingInput.less

```diff
@@ -9,13 +9,39 @@
   width: 100%;
 }
 
+.MessagingInput--InnerContainer--Content {
+  width: 100%;
+  .margin--right--xs();
+}
+
+.MessagingInput--Reply {
+  align-items: center;
+  justify-content: center;
+  .border--m(@neutral_silver);
+  border-bottom: none;
+  .padding--xs();
+}
+
 .MessagingInput--TextFieldLabel {
   color: @neutral_medium_gray;
   .margin--bottom--2xs();
 }
 
+.MessagingInput--TextFieldLabelHidden {
```

Comment:
Using this to hide the label so screen readers will stick pick up the label content

Reply:
Very nice!